src/main/app/pin-service/PinService.ts

```diff
@@ -9,18 +8,18 @@ export class PinService {
   constructor(private readonly idamApiAxios: AuthorizedAxios, private readonly logger: Logger) {}
 
   public loginWithPin = (clientId: string, redirectUri: string, pin: string) => {
-    const apiUrl = this.PIN_ENDPOINT + '?client_id=' + clientId + '&redirect_uri=' + redirectUri;
-
     return this.idamApiAxios
-      .get(apiUrl, {
+      .get(this.PIN_ENDPOINT + '?client_id=' + clientId + '&redirect_uri=' + redirectUri, {
```

Comment:
Shouldn't we be  passing the parameters in a "safer"  way, eg
```suggestion
      .get(this.PIN_ENDPOINT, {
          params: {
               client_id: clientId,
               redirect_uri:  redirectUri
          }
      }, {
```

Reply:
Also - there  might be an optional "state" parameter that we need to pass to idam-api if it's been sent to us.